include/pgduckdb/pg_declarations.hpp

```diff
@@ -10,13 +10,16 @@ It should not include any C++ code, only Postgres C declarations.
 */
 
 extern "C" {
-struct SnapshotData;
-typedef struct SnapshotData *Snapshot;
+typedef double Cardinality;
+
+struct Node;
```

Comment:
This is the only new one; I've just ordered the other declaration alphabetically for better readability/maintenance in the future.